crates/cli/src/subcommands/sql.rs

```diff
@@ -27,39 +28,31 @@ pub fn cli() -> clap::Command {
                 .conflicts_with("interactive")
                 .help("The SQL query to execute"),
         )
-        .arg(Arg::new("interactive")
-                 .long("interactive")
-                 .action(ArgAction::SetTrue)
-                 .conflicts_with("query")
-                 .help("Runs an interactive command prompt for `SQL` expressions"),)
+        .arg(
+            Arg::new("interactive")
+                .long("interactive")
+                .action(ArgAction::SetTrue)
+                .conflicts_with("query")
+                .help("Runs an interactive command prompt for `SQL` expressions"),
+        )
```

Comment:
I don't know why this reformat happened now. I was unable to retrigger it in a separate PR.